quicktile/commands.py

```diff
@@ -184,9 +184,9 @@ def call(self, command, winman, *args, **kwargs):
             return False
 
     def call_multiple(self, command, winman, *args, **kwargs):
-        # type: (List[str], WindowManager, *Any, **Any) -> bool
         """Resolve a textual positioning command and execute it.
            Accepts a comma seperated list as the command."""
+        # type: (str, WindowManager, *Any, **Any) -> bool
```

Comment:
I'll probably need another day or two.

If I'm seeing `List[str]` turn into `str` in type signatures without any explanation (whether it's a correction or a mistake), it suggests that, as a responsible project maintainer, I need to do a full audit before merging to make sure I wouldn't be letting in any new bugs.